symforce/codegen/backends/cpp/cpp_config.py

```diff
@@ -61,13 +70,12 @@ def templates_to_render(self, generated_file_name: str) -> T.List[T.Tuple[str, s
         return templates
 
     def printer(self) -> CodePrinter:
-        # NOTE(hayk): Is there any benefit to this being lazy?
-        from symforce.codegen.backends.cpp import cpp_code_printer
+        kwargs: T.Mapping[str, T.Any] = {"override_methods": self.override_methods}
```

Comment:
I think the linter might get mad about passing this as kwargs?  Not sure (CI will tell us).  If so the way to make it happy is to make this a `T.TypedDict`, but it might just be easier to just pass this both places below.

Reply:
Seems to be fine, I generally used T.Mapping for kwargs specifically because it's not invariant. Seems to pass CI.

Reply:
Ah right, it's because the value type is annotated as `T.Any` - if you left off the annotation and let it deduce the value type as `T.Optional[T.Dict[sympy.Function, str]]` it'd get mad unless you also used `TypedDict`